include/imu_ros2/adis1654x/adis1654x_data_access.h

```diff
@@ -0,0 +1,94 @@
+#ifndef ADIS1654X_DATA_ACCESS_H
+#define ADIS1654X_DATA_ACCESS_H
+
+// has delta channels
+#define ADIS_HAS_DELTA_BURST
+
+// has calibration scale channel
+#define ADIS_HAS_CLIB_SCALE
+
+#define ADIS_FLS_MEM_ENDURANCE 100000
+#define ADIS_MAX_SAMP_FREQ 4250.0
+
+// value to add to reg addr per page
+#define ADIS_PAGE_ID_VAL 0x80
+
+// global commands
+#define ADIS_GLOB_CMD_PAGE_ID 0x03
+#define ADIS_GLOB_CMD_ADDR_WITHOUT_PAGE 0x02
+#define ADIS_GLOB_CMD_ADDR \
+  (ADIS_PAGE_ID_VAL * ADIS_PT_OF_PERC_PAGE_ID + ADIS_GLOB_CMD_ADDR_WITHOUT_PAGE)
+
+#define ADIS_BIAS_CORRECTION_UPDATE_POS 0
+#define ADIS_SENSOR_SELF_TEST_POS 1
```

Comment:
use AIDS_SNSR_FAIL, and API is already available for reading this

Reply:
ADIS_SENSOR_SELF_TEST_POS might leave this way, and use AIDS_SNSR_FAIL for replacing ADIS_SENSOR_FAIL, this might be a more suitable, already existing API for it